PopPUNK/sparse_mst.py

```diff
@@ -106,8 +107,10 @@ def main():
         G = constructNetwork(rlist, rlist, None, 0,
                              sparse_input=sparse_mat, summarise=False)
         sys.stderr.write("Calculating MST (CPU)\n")
-
-    mst = generate_minimum_spanning_tree(G, args.gpu_graph)
+        # Generate minimum spanning tree
+        mst = generate_minimum_spanning_tree(G, args.gpu_graph)
+    
+    # Save output
```

Comment:
I think this change is incorrect -- did you test it with cugraph?

Reply:
I misread the cugraph code, you were right (just for the record)